runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/QueryResponse.java

```diff
@@ -0,0 +1,48 @@
+package org.corfudb.protocols.wireprotocol.orchestrator;
+
+import lombok.Getter;
+import org.corfudb.format.Types;
+
+import java.nio.ByteBuffer;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.nio.ByteBuffer'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

Reply:
done